public/app/plugins/datasource/azuremonitor/locales/en-US/grafana-azure-monitor-datasource.json

```diff
@@ -1,4 +1,177 @@
 {
+  "azuremonitor": {
```

Comment:
might want to exclude `azuremonitor` from the key generation, as its taken from parsing out the `app/plugins/datasource/{foo}` path